Controller/BaseController.php

```diff
@@ -0,0 +1,35 @@
+<?php
+namespace Myalpinerocks;
+
+abstract class BaseController
+{
+    public static function renderTemplate(string $templatePath, array $data = [])
```

Comment:
why static?

Reply:
My idea was to make all Controller classes static cos they do not have state (they do not keep information about entities like Entity classes do). Controller classes just do things as requested by user, like one-time jobs.

Reply:
`cos they do not have state` that doesn't mean they have to be static